test/integration/config/setupAfter.js

```diff
@@ -1,6 +1,6 @@
 // This file is for Jest-specific setup only and runs before our Jest tests.
 import { jestPreviewConfigure } from 'jest-preview';
-import '../config/assets/index.css';
+// import '../config/assets/index.css';
```

Comment:
Temporary fix for failing integration tests. Still need to figure out the timeout here